TODO.txt

```diff
@@ -38,6 +38,14 @@ Version 0.18
   ``neighbors`` and update the tests. Set the default value of ``connectivity``
   to 2.
 
+Version 0.18
+------------
+* In ``skimage.features.corner_peaks``, remove the warning.
+
+Version 0.18
+------------
+* In ``skimage.features.corner_peaks``, remove the warning.
+
```

Comment:
This is duplicated... More over, the section title `Version 0.18` is not necessary, simply add the item to the list.

Reply:
It's an issue with the merge. I'll fix it later if noone does.